pkg/model/stack.go

```diff
@@ -712,15 +698,19 @@ func LoadStack(name string, stackPaths []string, validate bool) (*Stack, error)
 	var resultStack *Stack
 
 	if len(stackPaths) == 0 {
-		stack, err := inferStack(name)
+		dir, err := os.Getwd()
+		if err != nil {
+			return nil, err
+		}
```

Comment:
This might have side effects regarding the working directory change we do when loading the manifest in okteto deploy

Reply:
This scares me a bit since `LoadStack` could have different result executing it before and after the change we do in the working directory. We rely too much on `os.Getwd` I think it doesn't affect as the previous `inferStack` function was relying also in the current working directory but we should be careful with that.